docs/developer/decision-records/2022-08-09-project-structure-review/README.md

```diff
@@ -0,0 +1,55 @@
+# Title of the Decision Record
+
+## Decision
+
+The project structure needs to be changed, to follow a "component" structure.
+
+## Rationale
+
+Currently it's hard for a newjoiner to understand the division between the different components of the application like 
+`control-plane`, `data-plane`, `federated-catalog` and so on...
+
+> A component is a particular EDC combination of dependencies that respond to a particular need, e.g.: `control-plane`, `data-plane`, `data-plane-selector`.
+This would help developers to extricate themselves in the codebase more easily and separate responsibilities in a better way.
```

Comment:
```suggestion
This would help developers to orient themselves in the codebase more easily and separate responsibilities in a better way.
```
"extricate" means "to remove oneself out of a particular situation"